app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    try:
+        for friend in friends:
+            Cafe.visit_cafe(cafe, visitor=friend)
```

Comment:
Instead of `Cafe.visit_cafe(cafe, visitor=friend)`, you should call the method directly on the `cafe` instance: `cafe.visit_cafe(visitor=friend)`. This ensures that the method is called on the correct instance of the `Cafe` class.